meta-resin-common/recipes-connectivity/connman/connman/allow_more_than_MAXNS_nameserver_entries_in_the_resolv_file.patch

```diff
@@ -0,0 +1,32 @@
+When using a DNS cache and resolver (like dnsmasq for example) we can have more than MAXNS entries
+in a custom resolv file (these entries can be the nameservers received from DHCP and/or added through
+connman's configuration files).
+
+We also tweak the order so that first will appear the nameservers set-up in connman's config files,
+followed by the nameservers received from DHCP in the order in which they have been received.
```

Comment:
I don't see exactly where you tweak this order in the patches. Can you please point me to the relevant change?